### DISCUSSION TITLE
Modal form not returning any value

### DISCUSSION THREAD

**Discussion by iin2022** [Budibase/budibase#6181] [General]
i have built a data entry screen that contains a lookup button that will open a modal screen and then after we click the select button the  from the the table ( in the modal screen ) it will close the modal screen and pass the selected value ( using app state ) to form field in data entry screen.
This was working in previous older version, but after recent update its not working anymore ?

Note if we switch from modal screen to full screen this working fine.
I have attached the sample project for this :

[modalsample-export-1654225632289.txt](https://github.com/Budibase/budibase/files/8829248/modalsample-export-1654225632289.txt)

**Comment by melohagan** [comment]
Hi @iin2022 

This is a related issue here: https://github.com/Budibase/budibase/issues/5629

Basically it doesn't work anymore because there was a change to default value. It only updates on the initial page load now.

There needs to be a redirect option on modal close, or some other new way to update field values from a screen modal.
